index.html

```diff
@@ -135,15 +135,12 @@ <h2>Feedback</h2>
 
 <h2>Contributing</h2>
 
-<p>Company is subject to the same
-  <a href="https://www.fsf.org/licensing/contributor-faq" target="_blank">copyright assignment</a>
-  policy as Emacs itself, org-mode, CEDET and other packages
-  in <a href="http://elpa.gnu.org/packages/">GNU ELPA</a>.
-
-  Any <a href="http://www.gnu.org/prep/maintain/html_node/Legally-Significant.html#Legally-Significant">legally
-  significant</a> contributions can only be accepted after the author has
-  completed their paperwork.  Please ask for the request form, and we'll send it to you.
-</p>
+<p>Company exists thanks to the input from many volunteers.</p>
```

Comment:
It's a good speech, but the live seems to overstate the case a little. Company, for good or bad, has been the work of individual effort for the most part. First Nikolaj (multiple versions and approaches tried over the years), then myself. With valuable contributions from others, but still.

We can swap "exists" for "is successful", how about that?

Reply:
Sure thing. I had something like "thrives" on my mind, but that was too much. Your suggestion seems to match much better the intended meaning and the actual state of things. ["input" covers bug reports and everything similar as well, btw.]

Reply:
> "input" covers bug reports and everything similar as well, btw

Yup.

Thank you. :+1: